packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -49,6 +49,18 @@
             </component>
             <!-- END ERROR BOUNDARY -->
 
+            <!-- Local Legend label -->
+            <!-- START ERROR BOUNDARY -->
+            <component
+                :is="errorBoundary"
+                tier="3">
+                <div
```

Comment:
I'm not convinced we need the div. Have you identified the issue of why this is needed?

Reply:
The SVG seems to have an invisible _bottom margin_, so I added a div with a set height to remove it

In the screenshots, I removed the set height and let it auto scale and you can see whitespace under the SVG
![image](https://user-images.githubusercontent.com/68010631/145018528-2d80e29e-77c6-4a0f-8ee5-e96504932da0.png)
![image](https://user-images.githubusercontent.com/68010631/145018569-e532a4f6-6f30-491f-83fc-a8dc122f3000.png)

Here it is with the set height SVG
![image](https://user-images.githubusercontent.com/68010631/145018903-bd9b277d-e52c-454b-9885-19de1aa3c5f1.png)

Reply:
It might be that the viewBox of the SVG is larger than the SVG causing this issue